packages/tldraw/src/lib/ui/components/Toolbar/ToggleToolLockedButton.tsx

```diff
@@ -29,8 +18,9 @@ export function ToggleToolLockedButton({ activeToolId }: ToggleToolLockedButtonP
 	const isToolLocked = useValue('is tool locked', () => editor.getInstanceState().isToolLocked, [
```

Comment:
Combine these two hooks into a single "show tool lock" value?

Reply:
We use `isToolLocked` to either show the `lock` or `unlock` icon, so we'd need this one anyway.